static/src/javascripts/projects/common/modules/experiments/tests/support-epic-circles.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import { getSync as getGeoLocation } from 'lib/geolocation';
+
+const targetUrl = () => {
+    const fromGeo = getGeoLocation();
+    if (fromGeo === 'US')
+        return 'https://support.theguardian.com/us/contribute';
+    if (fromGeo === 'GB')
+        return 'https://support.theguardian.com/uk/contribute';
+    return 'https://support.theguardian.com/';
```

Comment:
Very minor point, but maybe worth a comment to say that this condition shouldn't happen, given that the test is only UK & US - it slightly confused me when reading it

Reply:
Agreed, it _shouldn't_ happen. I just wondered if `fromGeo` could ever fail, in which case it's safer to assume a default state.